src/ncurses_ui.cc

```diff
@@ -238,6 +238,14 @@ default_colors = {
     { Color::Magenta, COLOR_MAGENTA },
     { Color::Cyan,    COLOR_CYAN },
     { Color::White,   COLOR_WHITE },
+    { Color::BrBlack,    8 },
```

Comment:
Lets either use number for all colors, or (what I think would be my preference), use `COLOR_BLAH + 8` for bright colors.